src/clr-angular/utils/i18n/strings.service.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class StringsService {
+  static replaceCommonString(source: string, key: string, value: string | number): string {
+    return source.replace(key, value.toString());
```

Comment:
I think @gnomeontherun was already touching on this but I think the new `ClrCommonStringsService`'s `parse` method can be used instead:

https://github.com/vmware/clarity/blob/d293c8f56f2853aeed0d6b0cf770d91a5534c2a7/src/clr-angular/utils/i18n/common-strings.service.ts#L34-L43

Reply:
Yeah, this was the  band-aid I used before his changes went in. I'm removing this service and will push the update soon.